apps/server/src/modules/learnroom/uc/dto/course-info.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { EntityId } from '@shared/domain/types';
+
+export class CourseInfoDto {
+	id: EntityId;
+
+	name: string;
+
+	teachers: string[];
+
+	classes: string[];
+
+	syncedWithGroup?: string;
```

Comment:
If this is an id then prefer to use EntityId, so others know that it is not a random string.